var/spack/repos/builtin/packages/msprime/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Msprime(PythonPackage):
```

Comment:
Any reason not to name this `py-msprime` like all other Python packages?

Reply:
As for tskit, it's a standalone tool so the py- isn't adapted.

https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name